nemo_skills/pipeline/utils/eval.py

```diff
@@ -28,6 +28,66 @@
 LOG = logging.getLogger(get_logger_name(__file__))
 
 
+def parse_eval_args(eval_args: str) -> tuple[str | None, dict]:
```

Comment:
this feels a bit brittle and hacky, but I can't currently think of a better way.. Maybe add a TODO to say that we ideally don't want to rely on custom parsing of the command, but some major refactoring or clever ideas might be needed :)